docs/Plugins/webhook.md

```diff
@@ -68,21 +73,56 @@ curl <devlake-host>/api/rest/plugins/webhook/1/deployments -X 'POST' -d '{
     "environment":"PRODUCTION",
     "repo_url":"https://github.com/apache/incubator-devlake/",
     "repo_id": "optional-repo-id",
+    "name": "optional-deployment-name",
     "ref_name": "optional-release-v0.17",
     "commit_sha":"015e3d3b480e417aede5a1293bd61de9b0fd051d",
+    "commit_msg":"optional-commit-message",
     "create_time":"2020-01-01T11:00:00+00:00",
     "start_time":"2020-01-01T12:00:00+00:00",
     "end_time":"2020-01-02T13:00:00+00:00",
     "result": "FAILURE"
   }'
 ```
+or if you want to push deployments to multiple repos in one request, you can use the following command:
```

Comment:
If you want to push a deployment that deploys on multi-repos (see the [discussion](https://github.com/apache/incubator-devlake/discussions/6162)), you can use the following command.